circle.yml

```diff
@@ -0,0 +1,27 @@
+machine:
+  node:
+    version: 7
+  pre:
+    - mkdir ~/.yarn-cache
+deployment:
+  staging:
+    branch: master
+    owner: artsy
+    commands:
+      - NODE_ENV=production make deploy env=staging
+  production:
+    branch: release
+    owner: artsy
+    commands:
+      - NODE_ENV=production make deploy env=production
```

Comment:
This matches with https://github.com/artsy/metaphysics/blob/master/circle.yml - should be 👍